### PR TITLE
[Support] Adds option to print payloads

### PR DESCRIPTION
**Pull request by nmuesch**
### What does this PR do?

Adds ability to specify `print_payloads` in the yaml file to have the Aggregator print out the metrics, service checks, and events its going to send right before we send them. 

### Motivation

Can be useful both in local developing as well as in troubleshooting support cases. Specifically made part of this to help debug a possible issue with the Disk check.

### Additional Notes

Right now its an option in the yaml file that won't be written out in the config_template.yaml file, can add it to a knowledge base article or the Docs in the Github repo though.

### CODE REVIEW

**Review by masci — CHANGES_REQUESTED** [review]
Let's add this option to the `config.go` module so we can keep track of it.
Also I'd personally s/print_payloads/log_payloads and put it around line 77 along with the other logging options.
No need to add it to the config file template.

File: pkg/config/config.go [hunk 1]
```diff
@@ -74,6 +74,7 @@ func init() {
 	Datadog.SetDefault("confd_dca_path", defaultDCAConfdPath)
 	Datadog.SetDefault("use_service_mapper", true)
 	Datadog.SetDefault("additional_checksd", defaultAdditionalChecksPath)
+	Datadog.SetDefault("log_payload", true)
```
Comment (masci) [inline pkg/config/config.go]: let's default to `false` or this is going to be spammy :)

Comment (masci) [reply pkg/config/config.go]: Also please don't hate me but can we call this `log_payloads`? 😊

**Review by masci — CHANGES_REQUESTED** [review]

File: pkg/config/config.go (hunk 1 above)
Comment (nmuesch) [reply pkg/config/config.go]: That should definitely be false! Good catch. Since I'm already fixing, I can definitely change the name haha.

**Review by masci — APPROVED** [review]

**Pull request merged**